openage/util/fslike/path.py

```diff
@@ -330,3 +348,16 @@ def mount(self, pathobj, priority=0) -> NoReturn:
         # pylint: disable=no-self-use,unused-argument
         # TODO: https://github.com/PyCQA/pylint/issues/2329
         raise PermissionError("Do not call mount on Path instances!")
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.delete()
```

Comment:
Be careful here. If you do this with a regular file (and not a temp file), this will also delete it!

You should either check if this is a tempfile or use `__del__` instead.